### DISCUSSION TITLE
how to create a start screen

### DISCUSSION THREAD

**Discussion by Baron-Samedy** [forntoh/LcdMenu#89] [Q&A] [answered]
Hello, how to create a startup splash screen so that the startup parameters are on the screen when loading, and when a key is pressed, a menu is displayed, and so that at the end of the execution of functions from the callback, the return goes to this startup screen without drawing the menu, perhaps there is such a function in the library?

**Comment by forntoh** [comment, marked as answer]
Check this 
https://github.com/forntoh/LcdMenu/discussions/85#discussioncomment-5953432
